src/model/scripts/hf_trainer/sbatch submit_deepspeed.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#SBATCH -p gpu
+#SBATCH -N 1 -c 64
+#SBATCH --ntasks-per-node=1
+#SBATCH --gpus=4
+#SBATCH -t 00:60:00
+#SBATCH -J test
+#SBATCH -A <project_id>
+
+
+module restore
+module load Miniconda3
+module load PrgEnv-gnu
+module load cpe-cuda
+module load cudatoolkit/22.7_11.7
+module load craype-accel-nvidia80
+module load aws-ofi-nccl
+
+conda deactivate
+conda activate <conda_prefix_path>
+
+module list
+
+export NCCL_DEBUG=INFO
+export NCCL_SOCKET_IFNAME=hsn 
+
+# apptainer exec  --bind  /project/lt200056-opgpth/stanford_alpaca:/home/nbuppodo --writable pytorch_image_fixed pip install git+https://github.com/huggingface/accelerate
+
+START=`date`
+starttime=$(date +%s)
+
+
+export WANDB_MODE="offline"
+
+torchrun --nproc_per_node=4 --master_port=3000 train.py \
+    --model_name_or_path /project/lt200056-opgpth/boss/stanford_alpaca/llama_2_7b_fixed_resized \
```

Comment:
Can you make sure this model is using thai tokenizer (The merged vocab one)

Reply:
already check